taskcluster/scripts/toolchain/android-gradle-dependencies/before.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -vex
```

Comment:
This is a copy of https://searchfox.org/mozilla-central/rev/2cd2d511c0d94a34fb7fa3b746f54170ee759e35/taskcluster/scripts/misc/android-gradle-dependencies/before.sh. I just `s/misc/toolchain/` and changed `/builds/worker/workspace/build` into `/builds/worker/checkouts/`